metastore/scripts/upgrade/hive/hive-schema-4.0.0.hive.sql

```diff
@@ -2008,6 +2008,75 @@ JOIN
 WHERE
   SE.SCHEDULED_QUERY_ID=SQ.SCHEDULED_QUERY_ID;
 
+CREATE OR REPLACE VIEW `TRANSACTIONS` (
```

Comment:
I do not see any permission checks. Please check how it's done for COMPACTIONS view. 
Also there is already a TRANSACTIONS view in sys.db, so no need for copy paste.